docs/docs.md

```diff
@@ -0,0 +1,628 @@
+
+## `GET`
+
+### Response
+
+```ts
+type Data = {
+  modeName: string
+  stats: (
+    | {
+        attempts: number
+        successes: number
+        statName: string
+        // 2018orwil
+        event: string
+        // qm1
+        match: string
+      }
+    | {
+        attempted: boolean
+        succeeded: boolean
+        statName: string
+        // 2018orwil
+        event: string
+        // qm1
+        match: string
+      })[]
+}[]
+```
+
+# `/event/{eventKey}/team/{team}/stats/teleop`
+
+## `GET`
+
+### Response
+
+```ts
+type Data = (
+  | {
+      attempts: number
+      successes: number
+      statName: string
+      // 2018orwil
+      event: string
+      // qm1
+      match: string
+    }
+  | {
+      attempted: boolean
+      succeeded: boolean
+      statName: string
+      // 2018orwil
+      event: string
+      // qm1
+      match: string
+    })[]
+```
+
+# `/event/{eventKey}/teams`
+
+## `GET`
+
+### Response
+
+```ts
+type Data = string[]
+```
+
+# `/events`
+
+## `GET`
+
+### Response
+
+```ts
+type Data = {
+  name: string
+  district?: string
+  week?: number
+  // UTC date
+  startDate: string
+  // UTC date
+  endDate: string
+  // Only if user is logged in
+  starred?: false
+  location: {
+    lat: number
+    lon: number
+  }
+}[]
+```
+
+# `/schema`
+
+## `GET`
+
+### Response
+
+```ts
+type Data = {
+  teleop: {
+    statName: string
+    statKey: string
+    type: "number" | "boolean" | "enum"
+  }[]
+  auto: {
+    statName: string
+    statKey: string
+    type: "number" | "boolean" | "enum"
+  }[]
+}
+```
+
+# `/team/{team}/automodes`
+
+## `GET`
+
+### Response
+
+```ts
+type Data = string[]
+```
+
+# `/team/{team}/stats/auto`
+
+## `GET`
+
+### Response
+
+```ts
+type Data = {
+  modeName: string
+  stats: (
+    | {
+        attempts: number
+        successes: number
+        statName: string
+        // 2018orwil
+        event: string
+        // qm1
+        match: string
+      }
+    | {
+        attempted: boolean
+        succeeded: boolean
+        statName: string
+        // 2018orwil
+        event: string
+        // qm1
+        match: string
+      })[]
+}[]
+```
+
+# `/team/{team}/stats/teleop`
+
+## `GET`
+
+### Response
+
+```ts
+type Data = {
+  team: string
+  teleop:
+    | {
+        attempts: {
+          max: number
+          avg: number
+        }
+        successes: {
+          max: number
+          avg: number
+        }
+        statName: string
+      }
+    | {
+        // total
+        attempts: number
+        // total
+        successes: number
+        statName: string
+      }
+  auto:
+    | {
+        attempts: {
+          max: number
+          avg: number
+        }
+        successes: {
+          max: number
+          avg: number
+        }
+        statName: string
+      }
+    | {
+        // total
+        attempts: number
+        // total
+        successes: number
+        statName: string
+      }
+}
+```
+
+# `/users/{userId}`
```

Comment:
needs comments about what admins can do vs normal users and what data admins get vs normal users